AstSemantics.md

```diff
@@ -256,14 +252,25 @@ one can arrange `block`s to put labels wherever one wants to jump to, except
 for one restriction: one can't jump into the middle of a loop from outside
 it. This restriction ensures the well-structured property discussed below.
 
-A `tableswitch` has a zero-based array of targets, a "default" target, an index
-operand, and a list of `case` nodes. It jumps to the target indexed in the
-array, or the default if the index is out of bounds. `tableswitch` targets may
-be either labels or `case` nodes.
+### Yielding values from control constructs
+
+The `if`, `br`, `br_if`, `case`, and `return` constructs do not yield values.
```

Comment:
How about saying that `if` and `br_if` yield an "empty" value (and define this as short for empty sequence of values), while the others abort their evaluation context, so do not yield anything at all (not even an empty value).